sponsors/models.py

```diff
@@ -819,3 +856,105 @@ def nullify(self, commit=True):
         if commit:
             self.sponsorship.save()
             self.save()
+
+
+########################################
+##### Benefit features abstract classes
+class BaseLogoPlacement(models.Model):
+    publisher = models.CharField(
+        max_length=30,
+        choices=[(c.value, c.name.replace("_", " ").title()) for c in PublisherChoices],
+        verbose_name="Publisher",
+        help_text="On which site should the logo be displayed?"
+    )
+    logo_place = models.CharField(
+        max_length=30,
+        choices=[(c.value, c.name.replace("_", " ").title()) for c in LogoPlacementChoices],
+        verbose_name="Logo Placement",
+        help_text="Where the logo should be placed?"
+    )
+
+    class Meta:
+        abstract = True
+
+
+######################################################
+##### SponsorshipBenefit features configuration models
+class BenefitFeatureConfiguration(PolymorphicModel):
+    """
+    Base class for sponsorship benefits configuration.
+    """
+    benefit = models.ForeignKey(SponsorshipBenefit, on_delete=models.CASCADE)
+
+    class Meta:
+        verbose_name = "Benefit Feature Configuration"
+        verbose_name_plural = "Benefit Feature Configurations"
+
+    @property
+    def benefit_feature_class(self):
+        """
+        Return a subclass of BenefitFeature related to this configuration.
+        Every configuration subclass must implement this property
+        """
+        raise NotImplementedError
+
+    def get_benefit_feature(self, **kwargs):
```

Comment:
For now this is 💯, but it may make sense to leave this as unimplemented and allow for each Configuration to make decisions, although of course they can be overridden in inheritance :)